src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.ts

```diff
@@ -125,20 +125,14 @@ export class RgwBucketListComponent extends ListWithDetails implements OnInit {
 
   transformBucketData() {
     _.forEach(this.buckets, (bucketKey) => {
-      const usageList = bucketKey['usage'];
       const maxBucketSize = bucketKey['bucket_quota']['max_size'];
       const maxBucketObjects = bucketKey['bucket_quota']['max_objects'];
-      let totalBucketSize = 0;
-      let numOfObjects = 0;
-      _.forEach(usageList, (usageKey) => {
-        totalBucketSize = totalBucketSize + usageKey.size_actual;
-        numOfObjects = numOfObjects + usageKey.num_objects;
-      });
-      bucketKey['bucket_size'] = totalBucketSize;
-      bucketKey['num_objects'] = numOfObjects;
-      bucketKey['size_usage'] = maxBucketSize > 0 ? totalBucketSize / maxBucketSize : undefined;
+      bucketKey['bucket_size'] = bucketKey['usage']['rgw.main']['size_actual'];
+      bucketKey['num_objects'] = bucketKey['usage']['rgw.main']['num_objects'];
```

Comment:
So we were doing these calculations in the front-end... 🙈 I guess our back-end for RGW is just a thin wrapper and the front-end simply swallows whatever comes from RGW. Surprisingly, we have everything properly typed in this component, except for the main data object which is of type... `object` 🤕 

I'd vote for generating proper `Bucket` and `Bucket[]` collection objects in the back-end, but then that wouldn't be a lightweight fix, so let's go with the easy fix then.